test/runtime/values.spec.js

```diff
@@ -133,4 +134,175 @@ describe('Values tests', function () {
             });
         });
     });
+
+    describe('string representation', function() {
+        it('toString', function() {
+            var tests = [
+                {
+                    value: null,
+                    expected: 'null'
+                },
+                {
+                    value: true,
+                    expected: 'true'
+                },
+                {
+                    value: false,
+                    expected: 'false'
+                },
+                {
+                    value: 5,
+                    expected: '5'
+                },
+                {
+                    value: Infinity,
+                    expected: 'Infinity'
+                },
+                {
+                    value: NaN,
+                    expected: 'NaN'
+                },
+                {
+                    value: '',
+                    expected: ''
+                },
+                {
+                    value: 'abcd',
+                    expected: 'abcd'
+                },
+                {
+                    value: new RegExp(),
+                    expected: '/(?:)/'
+                },
+                {
+                    value: /abcd/,
+                    expected: '/abcd/'
+                },
+                {
+                    value: new JuttleMoment.duration('5', 'seconds'),
+                    expected: '00:00:05.000'
+                },
+                {
+                    value: new JuttleMoment('2015-01-01T00:00:05'),
+                    expected: '2015-01-01T00:00:05.000Z'
+                },
+                {
+                    value: new Filter({}, 'a < 5'),
```

Comment:
I’d slightly prefer to pass the AST too (see e.g. [here](https://github.com/juttle/juttle/blob/master/test/compiler/inputs.spec.js#L92-L103)).